### ISSUE TITLE
Quadlet - podman_container ignores cpus parameter

### ISSUE THREAD

**Issue by jorti** [containers/ansible-podman-collections#857]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind bug

**Description**

`containers.podman.podman_container` ignores the `cpus` parameter when creating a quadlet file.

**Steps to reproduce the issue:**

1. Run a playbook that creates a container quadlet with a CPU limit using the `cpus` option.

**Describe the results you received:**

The `cpus` parameter is ignored:
```
$ cat ~/.config/containers/systemd/test.container 
[Container]
ContainerName=test
Image=registry.fedoraproject.org/fedora:40
```

**Describe the results you expected:**
I expect a line like this in the `.container` file:
```
PodmanArgs=--cpus=2.0
```

**Additional information you deem important (e.g. issue happens only occasionally):**

**Version of the `containers.podman` collection:**

```
containers.podman                        1.16.1
```

**Output of `ansible --version`:**

```
ansible [core 2.16.11]
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/juan/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.12/site-packages/ansible
  ansible collection location = /home/juan/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/bin/ansible
  python version = 3.12.6 (main, Sep  9 2024, 00:00:00) [GCC 14.2.1 20240801 (Red Hat 14.2.1-1)] (/usr/bin/python3)
  jinja version = 3.1.4
  libyaml = True
```

**Output of `podman version`:**

```
Client:       Podman Engine
Version:      5.2.3
API Version:  5.2.3
Go Version:   go1.22.7
Built:        Tue Sep 24 02:00:00 2024
OS/Arch:      linux/amd64
```

**Output of `podman info --debug`:**

``` yaml
host:
  arch: amd64
  buildahVersion: 1.37.3
  cgroupControllers:
  - cpu
  - io
  - memory
  - pids
  cgroupManager: systemd
  cgroupVersion: v2
  conmon:
    package: conmon-2.1.12-2.fc40.x86_64
    path: /usr/bin/conmon
    version: 'conmon version 2.1.12, commit: '
  cpuUtilization:
    idlePercent: 97.66
    systemPercent: 0.8
    userPercent: 1.54
  cpus: 24
  databaseBackend: sqlite
  distribution:
    distribution: fedora
    variant: workstation
    version: "40"
  eventLogger: journald
  freeLocks: 2048
  hostname: argon
  idMappings:
    gidmap:
    - container_id: 0
      host_id: 1000
      size: 1
    - container_id: 1
      host_id: 524288
      size: 65536
    uidmap:
    - container_id: 0
      host_id: 1000
      size: 1
    - container_id: 1
      host_id: 524288
      size: 65536
  kernel: 6.10.11-200.fc40.x86_64
  linkmode: dynamic
  logDriver: journald
  memFree: 52963823616
  memTotal: 67332497408
  networkBackend: netavark
  networkBackendInfo:
    backend: netavark
    dns:
      package: aardvark-dns-1.12.2-2.fc40.x86_64
      path: /usr/libexec/podman/aardvark-dns
      version: aardvark-dns 1.12.2
    package: netavark-1.12.2-1.fc40.x86_64
    path: /usr/libexec/podman/netavark
    version: netavark 1.12.2
  ociRuntime:
    name: crun
    package: crun-1.17-1.fc40.x86_64
    path: /usr/bin/crun
    version: |-
      crun version 1.17
      commit: 000fa0d4eeed8938301f3bcf8206405315bc1017
      rundir: /run/user/1000/crun
      spec: 1.0.0
      +SYSTEMD +SELINUX +APPARMOR +CAP +SECCOMP +EBPF +CRIU +LIBKRUN +WASM:wasmedge +YAJL
  os: linux
  pasta:
    executable: /usr/bin/pasta
    package: passt-0^20240906.g6b38f07-1.fc40.x86_64
    version: |
      pasta 0^20240906.g6b38f07-1.fc40.x86_64
      Copyright Red Hat
      GNU General Public License, version 2 or later
        <https://www.gnu.org/licenses/old-licenses/gpl-2.0.html>
      This is free software: you are free to change and redistribute it.
      There is NO WARRANTY, to the extent permitted by law.
  remoteSocket:
    exists: false
    path: /run/user/1000/podman/podman.sock
  rootlessNetworkCmd: pasta
  security:
    apparmorEnabled: false
    capabilities: CAP_CHOWN,CAP_DAC_OVERRIDE,CAP_FOWNER,CAP_FSETID,CAP_KILL,CAP_NET_BIND_SERVICE,CAP_SETFCAP,CAP_SETGID,CAP_SETPCAP,CAP_SETUID,CAP_SYS_CHROOT
    rootless: true
    seccompEnabled: true
    seccompProfilePath: /usr/share/containers/seccomp.json
    selinuxEnabled: true
  serviceIsRemote: false
  slirp4netns:
    executable: ""
    package: ""
    version: ""
  swapFree: 8589930496
  swapTotal: 8589930496
  uptime: 0h 38m 28.00s
  variant: ""
plugins:
  authorization: null
  log:
  - k8s-file
  - none
  - passthrough
  - journald
  network:
  - bridge
  - macvlan
  - ipvlan
  volume:
  - local
registries:
  search:
  - registry.fedoraproject.org
  - registry.access.redhat.com
  - docker.io
store:
  configFile: /home/juan/.config/containers/storage.conf
  containerStore:
    number: 0
    paused: 0
    running: 0
    stopped: 0
  graphDriverName: overlay
  graphOptions: {}
  graphRoot: /home/juan/.local/share/containers/storage
  graphRootAllocated: 1998678130688
  graphRootUsed: 886011604992
  graphStatus:
    Backing Filesystem: btrfs
    Native Overlay Diff: "true"
    Supports d_type: "true"
    Supports shifting: "false"
    Supports volatile: "true"
    Using metacopy: "false"
  imageCopyTmpDir: /var/tmp
  imageStore:
    number: 1
  runRoot: /run/user/1000/containers
  transientStore: false
  volumePath: /home/juan/.local/share/containers/storage/volumes
version:
  APIVersion: 5.2.3
  Built: 1727136000
  BuiltTime: Tue Sep 24 02:00:00 2024
  GitCommit: ""
  GoVersion: go1.22.7
  Os: linux
  OsArch: linux/amd64
  Version: 5.2.3
```

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

```
podman-5.2.3-1.fc40.x86_64
```

**Playbok you run with ansible (e.g. content of `playbook.yaml`):**

``` yaml
- name: Test
  hosts: localhost
  tasks:
    - name: Create container with CPU limit
      containers.podman.podman_container:
        state: quadlet
        name: test
        image: registry.fedoraproject.org/fedora:40
        cpus: "2.0"
```

**Command line and output of ansible run with high verbosity**

**Please NOTE: if you submit a bug about idempotency, run the playbook with `--diff` option, like:**

`ansible-playbook -i inventory --diff -vv playbook.yml`

```
$ ansible-playbook cpus.yml -vvvvv
ansible-playbook [core 2.16.11]
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/juan/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.12/site-packages/ansible
  ansible collection location = /home/juan/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/bin/ansible-playbook
  python version = 3.12.6 (main, Sep  9 2024, 00:00:00) [GCC 14.2.1 20240801 (Red Hat 14.2.1-1)] (/usr/bin/python3)
  jinja version = 3.1.4
  libyaml = True
Using /etc/ansible/ansible.cfg as config file
setting up inventory plugins
Loading collection ansible.builtin from 
host_list declined parsing /etc/ansible/hosts as it did not pass its verify_file() method
script declined parsing /etc/ansible/hosts as it did not pass its verify_file() method
auto declined parsing /etc/ansible/hosts as it did not pass its verify_file() method
Parsed /etc/ansible/hosts inventory source with ini plugin
[WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'
Loading collection containers.podman from /home/juan/.ansible/collections/ansible_collections/containers/podman
Loading callback plugin default of type stdout, v2.0 from /usr/lib/python3.12/site-packages/ansible/plugins/callback/default.py
Attempting to use 'default' callback.
Skipping callback 'default', as we already have a stdout callback.
Attempting to use 'junit' callback.
Attempting to use 'minimal' callback.
Skipping callback 'minimal', as we already have a stdout callback.
Attempting to use 'oneline' callback.
Skipping callback 'oneline', as we already have a stdout callback.
Attempting to use 'tree' callback.

PLAYBOOK: cpus.yml ********************************************************************************************************************************************
Positional arguments: cpus.yml
verbosity: 5
connection: ssh
become_method: sudo
tags: ('all',)
inventory: ('/etc/ansible/hosts',)
forks: 5
1 plays in cpus.yml

PLAY [Test] ***************************************************************************************************************************************************

TASK [Gathering Facts] ****************************************************************************************************************************************
task path: /tmp/test/cpus.yml:2
<127.0.0.1> ESTABLISH LOCAL CONNECTION FOR USER: juan
<127.0.0.1> EXEC /bin/sh -c 'echo ~juan && sleep 0'
<127.0.0.1> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/juan/.ansible/tmp `"&& mkdir "` echo /home/juan/.ansible/tmp/ansible-tmp-1727890811.3183994-13860-167933986335527 `" && echo ansible-tmp-1727890811.3183994-13860-167933986335527="` echo /home/juan/.ansible/tmp/ansible-tmp-1727890811.3183994-13860-167933986335527 `" ) && sleep 0'
Including module_utils file ansible/__init__.py
Including module_utils file ansible/module_utils/__init__.py
Including module_utils file ansible/module_utils/basic.py
Including module_utils file ansible/module_utils/_text.py
Including module_utils file ansible/module_utils/common/_json_compat.py
Including module_utils file ansible/module_utils/common/__init__.py
Including module_utils file ansible/module_utils/common/_utils.py
Including module_utils file ansible/module_utils/common/arg_spec.py
Including module_utils file ansible/module_utils/common/file.py
Including module_utils file ansible/module_utils/common/locale.py
Including module_utils file ansible/module_utils/common/parameters.py
Including module_utils file ansible/module_utils/common/collections.py
Including module_utils file ansible/module_utils/common/process.py
Including module_utils file ansible/module_utils/common/sys_info.py
Including module_utils file ansible/module_utils/common/text/converters.py
Including module_utils file ansible/module_utils/common/text/__init__.py
Including module_utils file ansible/module_utils/common/text/formatters.py
Including module_utils file ansible/module_utils/common/validation.py
Including module_utils file ansible/module_utils/common/warnings.py
Including module_utils file ansible/module_utils/compat/selectors.py
Including module_utils file ansible/module_utils/compat/__init__.py
Including module_utils file ansible/module_utils/compat/_selectors2.py
Including module_utils file ansible/module_utils/compat/selinux.py
Including module_utils file ansible/module_utils/distro/__init__.py
Including module_utils file ansible/module_utils/distro/_distro.py
Including module_utils file ansible/module_utils/errors.py
Including module_utils file ansible/module_utils/facts/ansible_collector.py
Including module_utils file ansible/module_utils/compat/typing.py
Including module_utils file ansible/module_utils/facts/__init__.py
Including module_utils file ansible/module_utils/facts/collector.py
Including module_utils file ansible/module_utils/facts/compat.py
Including module_utils file ansible/module_utils/facts/default_collectors.py
Including module_utils file ansible/module_utils/facts/hardware/aix.py
Including module_utils file ansible/module_utils/facts/hardware/__init__.py
Including module_utils file ansible/module_utils/facts/hardware/base.py
Including module_utils file ansible/module_utils/facts/hardware/darwin.py
Including module_utils file ansible/module_utils/facts/hardware/dragonfly.py
Including module_utils file ansible/module_utils/facts/hardware/freebsd.py
Including module_utils file ansible/module_utils/facts/hardware/hpux.py
Including module_utils file ansible/module_utils/facts/hardware/hurd.py
Including module_utils file ansible/module_utils/facts/hardware/linux.py
Including module_utils file ansible/module_utils/facts/hardware/netbsd.py
Including module_utils file ansible/module_utils/facts/hardware/openbsd.py
Including module_utils file ansible/module_utils/facts/hardware/sunos.py
Including module_utils file ansible/module_utils/facts/namespace.py
Including module_utils file ansible/module_utils/facts/network/aix.py
Including module_utils file ansible/module_utils/facts/network/__init__.py
Including module_utils file ansible/module_utils/facts/network/base.py
Including module_utils file ansible/module_utils/facts/network/darwin.py
Including module_utils file ansible/module_utils/facts/network/dragonfly.py
Including module_utils file ansible/module_utils/facts/network/fc_wwn.py
Including module_utils file ansible/module_utils/facts/network/freebsd.py
Including module_utils file ansible/module_utils/facts/network/generic_bsd.py
Including module_utils file ansible/module_utils/facts/network/hpux.py
Including module_utils file ansible/module_utils/facts/network/hurd.py
Including module_utils file ansible/module_utils/facts/network/iscsi.py
Including module_utils file ansible/module_utils/facts/network/linux.py
Including module_utils file ansible/module_utils/facts/network/netbsd.py
Including module_utils file ansible/module_utils/facts/network/nvme.py
Including module_utils file ansible/module_utils/facts/network/openbsd.py
Including module_utils file ansible/module_utils/facts/network/sunos.py
Including module_utils file ansible/module_utils/facts/other/facter.py
Including module_utils file ansible/module_utils/facts/other/__init__.py
Including module_utils file ansible/module_utils/facts/other/ohai.py
Including module_utils file ansible/module_utils/facts/sysctl.py
Including module_utils file ansible/module_utils/facts/system/apparmor.py
Including module_utils file ansible/module_utils/facts/system/__init__.py
Including module_utils file ansible/module_utils/facts/system/caps.py
Including module_utils file ansible/module_utils/facts/system/chroot.py
Including module_utils file ansible/module_utils/facts/system/cmdline.py
Including module_utils file ansible/module_utils/facts/system/date_time.py
Including module_utils file ansible/module_utils/compat/datetime.py
Including module_utils file ansible/module_utils/facts/system/distribution.py
Including module_utils file ansible/module_utils/facts/system/dns.py
Including module_utils file ansible/module_utils/facts/system/env.py
Including module_utils file ansible/module_utils/facts/system/fips.py
Including module_utils file ansible/module_utils/facts/system/loadavg.py
Including module_utils file ansible/module_utils/facts/system/local.py
Including module_utils file ansible/module_utils/facts/system/lsb.py
Including module_utils file ansible/module_utils/facts/system/pkg_mgr.py
Including module_utils file ansible/module_utils/facts/system/platform.py
Including module_utils file ansible/module_utils/facts/system/python.py
Including module_utils file ansible/module_utils/facts/system/selinux.py
Including module_utils file ansible/module_utils/facts/system/service_mgr.py
Including module_utils file ansible/module_utils/compat/version.py
Including module_utils file ansible/module_utils/facts/system/ssh_pub_keys.py
Including module_utils file ansible/module_utils/facts/system/user.py
Including module_utils file ansible/module_utils/facts/timeout.py
Including module_utils file ansible/module_utils/facts/utils.py
Including module_utils file ansible/module_utils/facts/virtual/base.py
Including module_utils file ansible/module_utils/facts/virtual/__init__.py
Including module_utils file ansible/module_utils/facts/virtual/dragonfly.py
Including module_utils file ansible/module_utils/facts/virtual/freebsd.py
Including module_utils file ansible/module_utils/facts/virtual/hpux.py
Including module_utils file ansible/module_utils/facts/virtual/linux.py
Including module_utils file ansible/module_utils/facts/virtual/netbsd.py
Including module_utils file ansible/module_utils/facts/virtual/openbsd.py
Including module_utils file ansible/module_utils/facts/virtual/sunos.py
Including module_utils file ansible/module_utils/facts/virtual/sysctl.py
Including module_utils file ansible/module_utils/parsing/convert_bool.py
Including module_utils file ansible/module_utils/parsing/__init__.py
Including module_utils file ansible/module_utils/pycompat24.py
Including module_utils file ansible/module_utils/six/__init__.py
Using module file /usr/lib/python3.12/site-packages/ansible/modules/setup.py
<127.0.0.1> PUT /home/juan/.ansible/tmp/ansible-local-13857jj3pylqp/tmp462rl8n1 TO /home/juan/.ansible/tmp/ansible-tmp-1727890811.3183994-13860-167933986335527/AnsiballZ_setup.py
<127.0.0.1> EXEC /bin/sh -c 'chmod u+x /home/juan/.ansible/tmp/ansible-tmp-1727890811.3183994-13860-167933986335527/ /home/juan/.ansible/tmp/ansible-tmp-1727890811.3183994-13860-167933986335527/AnsiballZ_setup.py && sleep 0'
<127.0.0.1> EXEC /bin/sh -c '/usr/bin/python3 /home/juan/.ansible/tmp/ansible-tmp-1727890811.3183994-13860-167933986335527/AnsiballZ_setup.py && sleep 0'
<127.0.0.1> EXEC /bin/sh -c 'rm -f -r /home/juan/.ansible/tmp/ansible-tmp-1727890811.3183994-13860-167933986335527/ > /dev/null 2>&1 && sleep 0'
ok: [localhost]

TASK [Create container with CPU limit] ************************************************************************************************************************
task path: /tmp/test/cpus.yml:5
<127.0.0.1> ESTABLISH LOCAL CONNECTION FOR USER: juan
<127.0.0.1> EXEC /bin/sh -c 'echo ~juan && sleep 0'
<127.0.0.1> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/juan/.ansible/tmp `"&& mkdir "` echo /home/juan/.ansible/tmp/ansible-tmp-1727890812.163453-13938-193954824357158 `" && echo ansible-tmp-1727890812.163453-13938-193954824357158="` echo /home/juan/.ansible/tmp/ansible-tmp-1727890812.163453-13938-193954824357158 `" ) && sleep 0'
Including module_utils file ansible/__init__.py
Including module_utils file ansible/module_utils/__init__.py
Including module_utils file ansible/module_utils/basic.py
Including module_utils file ansible/module_utils/_text.py
Including module_utils file ansible/module_utils/common/_json_compat.py
Including module_utils file ansible/module_utils/common/__init__.py
Including module_utils file ansible/module_utils/common/_utils.py
Including module_utils file ansible/module_utils/common/arg_spec.py
Including module_utils file ansible/module_utils/common/file.py
Including module_utils file ansible/module_utils/common/locale.py
Including module_utils file ansible/module_utils/common/parameters.py
Including module_utils file ansible/module_utils/common/collections.py
Including module_utils file ansible/module_utils/common/process.py
Including module_utils file ansible/module_utils/common/sys_info.py
Including module_utils file ansible/module_utils/common/text/converters.py
Including module_utils file ansible/module_utils/common/text/__init__.py
Including module_utils file ansible/module_utils/common/text/formatters.py
Including module_utils file ansible/module_utils/common/validation.py
Including module_utils file ansible/module_utils/common/warnings.py
Including module_utils file ansible/module_utils/compat/selectors.py
Including module_utils file ansible/module_utils/compat/__init__.py
Including module_utils file ansible/module_utils/compat/_selectors2.py
Including module_utils file ansible/module_utils/compat/selinux.py
Including module_utils file ansible/module_utils/distro/__init__.py
Including module_utils file ansible/module_utils/distro/_distro.py
Including module_utils file ansible/module_utils/errors.py
Including module_utils file ansible/module_utils/parsing/convert_bool.py
Including module_utils file ansible/module_utils/parsing/__init__.py
Including module_utils file ansible/module_utils/pycompat24.py
Including module_utils file ansible/module_utils/six/__init__.py
Including module_utils file ansible_collections/containers/podman/plugins/module_utils/podman/podman_container_lib.py
Including module_utils file ansible_collections/__init__.py
Including module_utils file ansible_collections/containers/__init__.py
Including module_utils file ansible_collections/containers/podman/__init__.py
Including module_utils file ansible_collections/containers/podman/plugins/__init__.py
Including module_utils file ansible_collections/containers/podman/plugins/module_utils/__init__.py
Including module_utils file ansible_collections/containers/podman/plugins/module_utils/podman/__init__.py
Including module_utils file ansible_collections/containers/podman/plugins/module_utils/podman/common.py
Including module_utils file ansible/module_utils/compat/version.py
Including module_utils file ansible_collections/containers/podman/plugins/module_utils/podman/quadlet.py
Using module file /home/juan/.ansible/collections/ansible_collections/containers/podman/plugins/modules/podman_container.py
<127.0.0.1> PUT /home/juan/.ansible/tmp/ansible-local-13857jj3pylqp/tmpj3o46u5a TO /home/juan/.ansible/tmp/ansible-tmp-1727890812.163453-13938-193954824357158/AnsiballZ_podman_container.py
<127.0.0.1> EXEC /bin/sh -c 'chmod u+x /home/juan/.ansible/tmp/ansible-tmp-1727890812.163453-13938-193954824357158/ /home/juan/.ansible/tmp/ansible-tmp-1727890812.163453-13938-193954824357158/AnsiballZ_podman_container.py && sleep 0'
<127.0.0.1> EXEC /bin/sh -c '/usr/bin/python3 /home/juan/.ansible/tmp/ansible-tmp-1727890812.163453-13938-193954824357158/AnsiballZ_podman_container.py && sleep 0'
<127.0.0.1> EXEC /bin/sh -c 'rm -f -r /home/juan/.ansible/tmp/ansible-tmp-1727890812.163453-13938-193954824357158/ > /dev/null 2>&1 && sleep 0'
ok: [localhost] => {
    "actions": [],
    "changed": false,
    "container": {},
    "invocation": {
        "module_args": {
            "annotation": null,
            "arch": null,
            "attach": null,
            "authfile": null,
            "blkio_weight": null,
            "blkio_weight_device": null,
            "cap_add": null,
            "cap_drop": null,
            "cgroup_conf": null,
            "cgroup_parent": null,
            "cgroupns": null,
            "cgroups": null,
            "chrootdirs": null,
            "cidfile": null,
            "cmd_args": null,
            "command": null,
            "conmon_pidfile": null,
            "cpu_period": null,
            "cpu_quota": null,
            "cpu_rt_period": null,
            "cpu_rt_runtime": null,
            "cpu_shares": null,
            "cpus": "2.0",
            "cpuset_cpus": null,
            "cpuset_mems": null,
            "debug": false,
            "decryption_key": null,
            "delete_depend": null,
            "delete_time": null,
            "delete_volumes": null,
            "detach": true,
            "detach_keys": null,
            "device": null,
            "device_cgroup_rule": null,
            "device_read_bps": null,
            "device_read_iops": null,
            "device_write_bps": null,
            "device_write_iops": null,
            "dns": null,
            "dns_option": null,
            "dns_search": null,
            "entrypoint": null,
            "env": null,
            "env_file": null,
            "env_host": null,
            "env_merge": null,
            "etc_hosts": null,
            "executable": "podman",
            "expose": null,
            "force_delete": true,
            "force_restart": false,
            "generate_systemd": {},
            "gidmap": null,
            "global_args": [],
            "gpus": null,
            "group_add": null,
            "group_entry": null,
            "health_startup_cmd": null,
            "health_startup_interval": null,
            "health_startup_retries": null,
            "health_startup_success": null,
            "health_startup_timeout": null,
            "healthcheck": null,
            "healthcheck_failure_action": null,
            "healthcheck_interval": null,
            "healthcheck_retries": null,
            "healthcheck_start_period": null,
            "healthcheck_timeout": null,
            "hooks_dir": null,
            "hostname": null,
            "hostuser": null,
            "http_proxy": null,
            "image": "registry.fedoraproject.org/fedora:40",
            "image_strict": false,
            "image_volume": null,
            "init": null,
            "init_ctr": null,
            "init_path": null,
            "interactive": null,
            "ip": null,
            "ip6": null,
            "ipc": null,
            "kernel_memory": null,
            "label": null,
            "label_file": null,
            "log_driver": null,
            "log_level": null,
            "log_opt": null,
            "mac_address": null,
            "memory": null,
            "memory_reservation": null,
            "memory_swap": null,
            "memory_swappiness": null,
            "mount": null,
            "name": "test",
            "network": null,
            "network_aliases": null,
            "no_healthcheck": null,
            "no_hosts": null,
            "oom_kill_disable": null,
            "oom_score_adj": null,
            "os": null,
            "passwd": null,
            "passwd_entry": null,
            "personality": null,
            "pid": null,
            "pid_file": null,
            "pids_limit": null,
            "platform": null,
            "pod": null,
            "pod_id_file": null,
            "podman_args": [],
            "preserve_fd": null,
            "preserve_fds": null,
            "privileged": null,
            "publish": null,
            "publish_all": null,
            "pull": null,
            "quadlet_dir": null,
            "quadlet_filename": null,
            "quadlet_options": null,
            "rdt_class": null,
            "read_only": null,
            "read_only_tmpfs": null,
            "recreate": false,
            "requires": null,
            "restart_policy": null,
            "restart_time": null,
            "retry": null,
            "retry_delay": null,
            "rm": null,
            "rmi": null,
            "rootfs": null,
            "sdnotify": null,
            "seccomp_policy": null,
            "secrets": null,
            "security_opt": null,
            "shm_size": null,
            "shm_size_systemd": null,
            "sig_proxy": null,
            "state": "quadlet",
            "stop_signal": null,
            "stop_time": null,
            "stop_timeout": null,
            "subgidname": null,
            "subuidname": null,
            "sysctl": null,
            "systemd": null,
            "timeout": null,
            "timezone": null,
            "tls_verify": null,
            "tmpfs": null,
            "tty": null,
            "uidmap": null,
            "ulimit": null,
            "umask": null,
            "unsetenv": null,
            "unsetenv_all": null,
            "user": null,
            "userns": null,
            "uts": null,
            "variant": null,
            "volume": null,
            "volumes_from": null,
            "workdir": null
        }
    }
}

PLAY RECAP ****************************************************************************************************************************************************
localhost                  : ok=2    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

Bare metal host.

**Comment by sshnaidm** [comment]
I believe it's being solved in #847

**Comment by sshnaidm** [comment]
Fixed in https://github.com/containers/ansible-podman-collections/pull/847
